main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,33 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  var doAddColumnWithUrlOfMatchedEntities=function(){
+    var serviceUrl = null;
+    var service = null;
+    if (column.reconConfig) {
+        serviceUrl = column.reconConfig.service;
+    }
+    if (serviceUrl) {
+        service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+    } 
+    if(service&&service.view)
+    {
+    Refine.postCoreProcess(
+      "add-column", 
+      {
+        baseColumnName: column.name,  
+        newColumnName: "Url of Matched entities", 
+        columnInsertIndex: columnIndex + 1,
+        onError: "set-to-blank"
+      },
+      { expression: 'if(cell.recon.match!=null,"' + service.view.url + '".replace("{{id}}",escape(cell.recon.match.id,"url")),"")' },
+      { modelsChanged: true },
+      
+    );
+    }
+    else
+    alert($.i18n('core-views/service-does-not-associate-URLs-to-the-entities-it-contains'));
```

Comment:
We generally enclose all `if` or `else` blocks with braces even if it's not needed, to avoid accidents, so I would do it for this `else` block too.